extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -122,15 +119,33 @@ describes.realWin('amp-bind', {
   it('should scan for bindings when ampdoc is ready', () => {
     createElementWithBinding('[onePlusOne]="1+1"');
     expect(bind.boundElements_.length).to.equal(0);
-    return onBindReady(() => {
+    return onBindReady().then(() => {
       expect(bind.boundElements_.length).to.equal(1);
     });
   });
 
+  it('should have same state after removing + re-adding a subtree', () => {
```

Comment:
IMO the method invocations on the evaluator are more of an implementation detail of the Bind object.